api/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -24,6 +24,21 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
```

Comment:
small doubt, do we actually use the deepcopy?

Reply:
I don't think we do